contracts/auxiliar/CommissionSplitter.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity 0.5.8;
+
+import "moc-governance/contracts/Governance/Governed.sol";
+import "moc-governance/contracts/Governance/IGovernor.sol";
+import "../MoC.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-eth/contracts/utils/ReentrancyGuard.sol";
+
+/**
+  @dev Contract that split his balance between two addresses based on a
+  proportion defined by Governance. One of those addresses should
+  be a Money on Chain main contract.
+ */
+contract CommissionSplitter is Governed, ReentrancyGuard {
+    event SplitExecuted(uint256 commissionAmount, uint256 mocAmount);
+    // Math
+    using SafeMath for uint256;
+    uint256 public constant PRECISION = 10**18;
+
+    // Final receiver address
+    address payable public commissionsAddress;
+    // Proportion of the balance to send to moc
+    uint256 internal mocProportion;
```

Comment:
any reason why not making this be public? Kind of make the `split()` less predictable (or at least id demands some extra steps to find the value)